### DISCUSSION TITLE
Welcome to mercurius Discussions!

### DISCUSSION THREAD

**Discussion by mcollina** [mercurius-js/mercurius#575] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by smolinari** [comment]
Just want to say, thanks so much for this package! So glad there is this good alternative to Apollo.....

Scott

**Comment by jonnydgreen** [comment]
Hey everyone, I'm Jonny - I'm a physicist come software engineer and in my spare time I like to do lots of running!

I have been contributing to Mercurius for almost a year now and have also been involved with the `mercurius-auth` and `mercurius-validation` plugins. Thank you to everyone in the community for being so welcoming, it has been a lot of fun to contribute to! :)

**Comment by victororlyk** [comment]
Hi, is there a way we can enable playground instead of graphiql?
I seen that there was merged pr but do not see the optino of 'playground ' 
https://github.com/mercurius-js/mercurius/issues/52

**Reply by smolinari** [reply]
@victororlyk - It would be better if you open a new discussion thread.

Scott
